<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Send">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate QR Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>6</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="DOB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>6</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DOB</string>
    </property>
   </widget>
   <widget class="QLabel" name="Medicine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>106</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Medication Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="NameTextBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="DOBTextBOX">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="InstructionBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Description">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>196</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Instruction</string>
    </property>
   </widget>
   <widget class="QLabel" name="Time">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>106</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DosageBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Medicine_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>106</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dosage</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="FrequencyBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DOB_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>6</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PPSN</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="PPSNBOX">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="qrcodeimage">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>341</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Duration">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>106</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DurationBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="MedicineTextBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Load">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load into Array</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonClear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCareTaker">
    <property name="title">
     <string>TakingCare</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuQR_Reader">
    <property name="title">
     <string>QR Reader</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuCareTaker"/>
   <addaction name="menuQR_Reader"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Send</sender>
   <signal>clicked()</signal>
   <receiver>InstructionBox</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Send</sender>
   <signal>clicked(bool)</signal>
   <receiver>DOBTextBOX</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Send</sender>
   <signal>clicked(bool)</signal>
   <receiver>NameTextBox</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ButtonClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ButtonClicked()</slot>
  <slot>ButtonClear()</slot>
 </slots>
</ui>
